Resources/Textures/Clothing/OuterClothing/Armor/bulletproof.rsi/meta.json

```diff
@@ -1,26 +1,26 @@
 {
-  "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from cev-eris at commit https://github.com/discordia-space/CEV-Eris/commit/760f0be7af33a31f5a08a3291864e91539d0ebb7",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "icon"
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/dbc2435fa562f4ef130a7112d2a4cc9c80099894",
```

Comment:
Since the armor is modified, did you make modifications? If so you can also mention "Modified by USERNAME for SS14" or something similar here.

Reply:
Solved. Copyright has been fixed.